test/MetadataGenerator/_files/Chapter-csl.json

```diff
@@ -1 +1 @@
-[{"type":"chapter","author":[{"given":"Stephen F.","family":"Ackley"},{"given":"M A","family":"Lange"},{"given":"Peter","family":"Wadhams"}],"editor":[{"given":"Stephen F.","family":"Ackley"},{"given":"Wilford F","family":"Weeks"}],"issued":{"raw":"1990-02-01"},"issue":"1","number-of-pages":6,"volume":"90","abstract":"In model simulations of seasonal pack-ice growth, snow cover is treated as insulating layer that inhibits ice growth, but, during field work in 1986, it was found that several factors negate this predicted behaviour. Estimates that snow cover increases sea-ice thickness by 20 to 30% over model predictions by flooding and infiltration mechanisms.","container-title":"Sea ice properties and processes \u2013 Proceedings of the W.F. Weeks Sea Ice Symposium","page":"16-21","page-first":16,"publisher":"U.S. Army Corps of Engineers, Cold Regions Research & Engineering Laboratory","publisher-place":"Hanover","title":"Snow cover effects on Antarctic sea ice thickness","URL":"https:\/\/www.coldregions.org\/vufind\/Record\/120367","language":"en"}]
\ No newline at end of file
+[{"type":"chapter","author":[{"given":"Stephen F.","family":"Ackley"},{"given":"M A","family":"Lange"},{"given":"Peter","family":"Wadhams"}],"editor":[{"given":"Stephen F.","family":"Ackley"},{"given":"Wilford F","family":"Weeks"}],"issued":{"raw":"1990-02-01"},"issue":"1","number-of-pages":6,"volume":"90","container-title":"Sea ice properties and processes \u2013 Proceedings of the W.F. Weeks Sea Ice Symposium","page":"16-21","page-first":16,"publisher":"U.S. Army Corps of Engineers, Cold Regions Research & Engineering Laboratory","publisher-place":"Hanover","title":"Snow cover effects on Antarctic sea ice thickness","URL":"https:\/\/www.coldregions.org\/vufind\/Record\/120367","language":"en"}]
```

Comment:
Please don't use any real data, especially nothing identifiable. You can use anything that is in Application test data anyway, but even that does not seem necessary for Unit Testing.